controllers/integrationpipeline/integrationpipeline_controller.go

```diff
@@ -133,13 +118,12 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 
 // AdapterInterface is an interface defining all the operations that should be defined in an Integration adapter.
 type AdapterInterface interface {
-	EnsureSnapshotExists() (controller.OperationResult, error)
 	EnsureSnapshotPassedAllTests() (controller.OperationResult, error)
 	EnsureStatusReported() (controller.OperationResult, error)
 	EnsureEphemeralEnvironmentsCleanedUp() (controller.OperationResult, error)
 }
 
-// SetupController creates a new Integration controller and adds it to the Manager.
+// SetupController creates a new Integration reconciler and adds it to the Manager.
```

Comment:
https://github.com/redhat-appstudio/integration-service/pull/254/files in this PR, reconciler was replaced by controller, here you are doing it in the opposite way.

Also why do you have to do this extra change in this already big PR?

Reply:
IMO you answered to wrong thread, but I got it

Reply:
Don't know why git consider this `integrationpipeline` as a renamed folder of `pipeline`.

Reply:
becuase marking it as moved instead of deleted results into less diff lines

Reply:
Thanks. I guessed so. `Git` is so smart.